.github/workflows/backend.yml

```diff
@@ -344,6 +344,7 @@ jobs:
         migration,
         monolith-dbs,
         typing,
+        check-rpc-for-breaking-changes,
```

Comment:
these have to be in the same file

Reply:
Oops, stray edit. Will fix when I move the job.